python/cucim/setup.py

```diff
@@ -49,14 +44,13 @@ def read(*names, **kwargs):
     url='https://github.com/rapidsai/cucim',
     packages=find_packages('src'),
     package_dir={'cucim': 'src/cucim'},
-    py_modules=[splitext(basename(path))[0] for path in glob('src/*.py')],
+    # # https://docs.python.org/3/distutils/setupscript.html
```

Comment:
Any reason to leave these comments here instead of removing ?  

Same question for the `package_data=` commented code below

Reply:
Oh, no specific reason. Just for future reference. Will remove it. Thank you for pointing out that!